crates/rpc/rpc-eth-api/Cargo.toml

```diff
@@ -21,7 +21,7 @@ reth-evm.workspace = true
 reth-primitives.workspace = true
 reth-provider.workspace = true
 reth-revm.workspace = true
-reth-rpc-types.workspace = true
+# reth-rpc-types.workspace = true
```

Comment:
cool, can we remove this entirely?

Reply:
yes after I rebased seems like it's not used anymore at all- will have a go at removing entirely

Reply:
@mattsse I feel like this reth-rpc-types doesn't do much anymore, should I give it a go to remove it fully ? (in this or separate pr)?

Reply:
yep: https://github.com/paradigmxyz/reth/issues/11198